pkgs/by-name/sy/syncat/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, rustPlatform, fetchFromGitHub, openssl, libgit2, pkg-config }:
```

Comment:
Very nitty, but these should be ordered by usage (last point): https://nixos.org/manual/nixpkgs/stable/#sec-syntax

Reply:
no, these have no established order, see #247424

Reply:
Okay, that's news to me. I need to start checking the unstable manual... Thanks for the heads up!

Ignore my previous comment. Feel free to mark as resolved.

Reply:
I mark it as resolved because of the merged pr removing this statement (: